src/constants/endpointsPages.ts

```diff
@@ -160,9 +160,9 @@ export const ENDPOINTS_PAGES_RPC: EndpointsPagesProps = {
         docsUrl:
           "https://developers.stellar.org/docs/data/rpc/api-reference/methods/sendTransaction",
         docsLabel: "sendTransaction",
-        endpointUrlTemplate: "/{?transaction}",
+        endpointUrlTemplate: "{?tx}",
         requestMethod: "POST",
-        requiredParams: "/{?transaction}",
+        requiredParams: "/{?tx}",
```

Comment:
For sendTransactions, the tx envelope is required. https://developers.stellar.org/docs/data/rpc/api-reference/methods/sendTransaction

Reply:
fixed!